store/views.py

```diff
@@ -77,6 +87,47 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    response_data = {
+        'message' : 'failure',
+    }
+
+    copy_id = request.POST.get('bid')
+    copy = BookCopy.objects.get(id = copy_id)
+    copy.status = True
+    copy.borrower = None
+    copy.borrow_date = None
+    copy.save();
+
+    response_data['message'] = 'success'
+    return JsonResponse(response_data)
+
+@csrf_exempt
+@login_required
+def rateBookView(request, bid) :
+    if (request.method == 'POST') :
+        rating = int(request.POST.get('rating'))
+        book = Book.objects.get(id=bid)
+        
+        if (Ratings.objects.filter(book=book, user=request.user).exists()) :
+            ratedBy = len(Ratings.objects.filter(book=book).all())
```

Comment:
`ratedBy` here would be `1` since each user can rate a book only once, right? (changes are allowed)